index.html

```diff
@@ -2,9 +2,37 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>02-Box-Model</title>
+    <link rel="stylesheet" href="./app.css">
   </head>
   <body>
+    <div class="navbar">
+      <h1 class="nav_bar-left">Popular Github</h1>
+
+      <h2 class="nav_bar-right">People</h2>
+
+      <h2 class="nav_bar-right">Repos</h2>
+    </div>
+
+    <div class="user_image">
+      <h1 class="name">Dustin Walker</h1>
+        <h2 class="user_name">DustinJWalker</h2>
+        <h3 class="list_of_repos">02-box-model</h3>
+        <h3 class="list_of_repos">01-html-introduction</h3>
+        <div class="repo_dot"></div>
```

Comment:
I would recommend moving the `repo_dot` within the element for `repo_lang_name`.
Also thinking of an outline the language should likely be grouped with the repo name that the language describes on.

Another tip here would be to use a text based tag `h1-6`/`p`/etc to wrap the "JS" string.
It is best to avoid putting plain text or inline style elements directly within a div.